access_point/server/server.py

```diff
@@ -16,28 +16,35 @@ def index():
 def install():
     return 'starting the installation script'
 
-@app.route('/config/<stt>/<tts>/<hotword>/<wake>')
-def config(stt, tts, hotword, wake):
-    subprocess.call(['sudo', 'bash' , access_point_folder + '/config.sh', stt, tts, hotword, wake])  #nosec #pylint-disable type: ignore
-    display_message = {"Configuration":"Successful"}
+@app.route('/config',methods=['GET'])
+def config():
+    stt = request.args.get('stt')
+    tts = request.args.get('tts')
+    hotword = request.args.get('hotword')
+    wake = request.args.get('wake')
+    subprocess.call(['sudo bash ' + access_point_folder + '/config.sh ' + ' {} {} {} {}  & '.format(stt, tts, hotword, wake)],shell=True)  #nosec #pylint-disable type: ignore
+    display_message = {"configuration":"successful", "stt": stt, "tts": tts, "hotword": hotword, "wake":wake}
     resp = jsonify(display_message)
     resp.status_code = 200
     return resp # pylint-enable
 
-@app.route('/auth/<auth>/<email>/<passwd>')
-def login(auth, email, passwd):
-    subprocess.call(['sudo', 'bash', access_point_folder + '/login.sh', auth, email, passwd]) #nosec #pylint-disable type: ignore
-    display_message = {"Authentication":"Successful"}
+@app.route('/auth',methods=['GET'])
+def login():
+    auth = request.args.get('auth')
+    email = request.args.get('email')
+    password = request.args.get('password')
+    subprocess.call(['sudo', 'bash', access_point_folder + '/login.sh', auth, email, password]) #nosec #pylint-disable type: ignore
+    display_message = {"authentication":"successful", "auth": auth, "email": email, "password": password}
     resp = jsonify(display_message)
     resp.status_code = 200
     return resp # pylint-enable
 
-@app.route('/wifi_credentials/<wifissid>/<wifipassd>')
-def wifi_config(wifissid,wifipassd):
-    wifi_ssid = wifissid
-    wifi_password = wifipassd
+@app.route('/wifi_credentials',methods=['GET'])
```

Comment:
Python coding style, please!

Reply:
@hongquan , I don't understand

Reply:
https://www.python.org/dev/peps/pep-0008/?#whitespace-in-expressions-and-statements